packages/quantic/force-app/main/default/lwc/quanticFeedbackModalQna/__tests__/quanticFeedbackModalQna.test.js

```diff
@@ -0,0 +1,299 @@
+import QuanticFeedbackModalQna from 'c/quanticFeedbackModalQna';
+import {createElement} from 'lwc';
+
+jest.mock('@salesforce/label/c.quantic_Yes', () => ({default: 'Yes'}), {
+  virtual: true,
+});
+jest.mock('@salesforce/label/c.quantic_No', () => ({default: 'No'}), {
+  virtual: true,
+});
+jest.mock(
+  '@salesforce/label/c.quantic_FeedbackNotSure',
+  () => ({default: 'Not sure'}),
+  {
+    virtual: true,
+  }
+);
+
+const functionsMocks = {
+  handleSubmit: jest.fn(),
+};
+
+const selectors = {
+  answerEvaluationQuestions: 'lightning-radio-group',
+  questionTopicRadio: 'lightning-radio-group[data-name="correctTopic"]',
+  questionHallucinationRadio:
+    'lightning-radio-group[data-name="hallucinationFree"]',
+  questionDocumentedRadio: 'lightning-radio-group[data-name="documented"]',
+  questionReadableRadio: 'lightning-radio-group[data-name="readable"]',
+  documentUrlInput: 'lightning-input[data-name="documentUrl"]',
+  additionalDetailsTextarea: 'lightning-textarea[data-name="details"]',
+  skipButton: 'lightning-button.skip',
+  submitButton: 'lightning-button.submit-feedback',
+  successMessage: '.feedback-modal-body__success-message',
+};
+
+const defaultOptions = {
+  liked: false,
+  handleSubmit: functionsMocks.handleSubmit,
+};
+
+function createTestComponent(options = defaultOptions) {
+  const element = createElement('c-quantic-feedback-modal-qna', {
+    is: QuanticFeedbackModalQna,
+  });
+
+  for (const [key, value] of Object.entries(options)) {
+    element[key] = value;
+  }
+
+  document.body.appendChild(element);
+  return element;
+}
+
+// Helper function to wait until the microtask queue is empty.
+function flushPromises() {
+  return new Promise((resolve) => setTimeout(resolve, 0));
+}
+
+function lightningElementValueChange(element, selector, value) {
+  const lightningElement = element.shadowRoot.querySelector(selector);
+  // All base components in Jest tests are stubs. So we have to set the value manually.
+  // Including to the checkValidity method.
```

Comment:
I think this comment should be adjusted,

Reply:
To what?

Reply:
It's just that `Including to the checkValidity method` doesn't seem like a correct english sentence x)